src/transform-sdk/tests/admin_client.go

```diff
@@ -192,6 +193,27 @@ func (cl *AdminAPIClient) DeleteTransform(ctx context.Context, name string) erro
 	return checkResponse(resp)
 }
 
+// SetLog level for a broker logger
+func (cl *AdminAPIClient) SetLogLevel(ctx context.Context, logger string, level string) error {
+	endpoint, err := url.JoinPath(cl.baseUrl, "/v1/config/log_level", url.PathEscape(logger))
+	if err != nil {
+		return fmt.Errorf("failed to join url path: %w", err)
+	}
+	endpoint += fmt.Sprintf("?level=%s", level)
```

Comment:
This will only temporarily set the log level... if you want to have it not expire you need to set `expires=0`

Reply:
Oh additionally for `trace` and `debug` levels you can't set `expires=0` without also specifying `force=true`.  https://redpandadata.atlassian.net/wiki/spaces/CORE/pages/511148037/Verbose+Logging+Maximum+Timeout

Reply:
Yeah that's fine, it was enough to debug the test, this is really here in case I ever need to do it again